ext/broadcast_channel/lib.rs

```diff
@@ -110,6 +110,8 @@ pub fn init<BC: BroadcastChannel + 'static>(
   unstable: bool,
 ) -> Extension {
   Extension::builder()
+    .name("broadcast channel")
```

Comment:
suggest to match crate name: `Extension::builder("deno_broadcast_channel")` it will make it more obvious that it's a built-in extension